webui/react/src/components/kit/Icon.tsx

```diff
@@ -117,11 +256,28 @@ const Icon: React.FC<Props> = ({
 }: Props) => {
   const classes = [css.base];
 
-  if (name) classes.push(`icon-${name}`);
+  const svgIcon = useMemo(() => {
+    if (name === 'columns') return <ColumnsIcon color={color} size={size} title={title} />;
+    if (name === 'filter') return <FilterIcon color={color} size={size} title={title} />;
+    if (name === 'options') return <OptionsIcon color={color} size={size} title={title} />;
+    if (name === 'panel') return <PanelIcon color={color} size={size} title={title} />;
+    if (name === 'row-small') return <RowIconSmall color={color} size={size} title={title} />;
+    if (name === 'row-medium') return <RowIconMedium color={color} size={size} title={title} />;
+    if (name === 'row-large') return <RowIconLarge color={color} size={size} title={title} />;
+    if (name === 'row-xl') return <RowIconExtraLarge color={color} size={size} title={title} />;
+
+    return null;
+  }, [color, name, size, title]);
+
+  if (name && name !== 'filter') classes.push(`icon-${name}`);
   if (size) classes.push(css[size]);
   if (color) classes.push(css[color]);
 
-  const icon = <span aria-label={title} className={classes.join(' ')} />;
+  const icon = (
+    <span aria-label={title} className={classes.join(' ')}>
+      {svgIcon}
+    </span>
```

Comment:
can we avoid applying the `size` and `color` classes twice with svg icons now that we're always using the span?